grid/chat/messages/profile.supr

```diff
@@ -0,0 +1,278 @@
+name = "chat/messages"
+version = "1.0.0"
+
+"""
+Get chat messages
+Retrieve paginated list of messages
+"""
+usecase GetMessages {
+	input {
+		"""
+		Destination
+		Representation of a messaging destination. Can be ID or channel name, depending on the provider capability.
+		"""
+		destination! string!
+		
+		"""
+		After Date
+		Retrieve only messages after this date and time, UNIX timestamp format (e.g. 1095379198.75) 
+		"""
+		afterDate number
+		
+		"""
+		Before Date
+		Retrieve only messages before this date and time, UNIX timestamp format (e.g. 1095379198.75)
+		"""
+		beforeDate number
+
+		"""
+		Limit
+		Maximum number of messages to retrieve in one page
+		"""
+		limit number
+		
+		"""
+		Page
+		Identification of page with paginated results (cursor)
+		"""
+		page string!
+	}
+	
+	result Messages
+	error Error
+
+	example Successful {
+		input {
+			destination = 'general',
+			limit = 2
+		}
+
+		result {
+			messages = [
+				{
+					id = '1'
+					author = {
+						id = 'U01'
+						username = 'user1'
+					},
+					createdAt = 1643835553.658259,
+					text = 'test 1'
+				}
+				{
+					id = '2'
+					author = {
+						id = 'U02'
+						username = 'user2'
+					},
+					createdAt = 1643835555.648069
+					text = 'test 2'
+				}
+			]
+		}
+	}
+
+	example Failed {
+		input {
+			destination = 'not-existing',
+		}
+
+		error {
+			title = 'channel_not_found'
+		}
+	}
+}
+
+model Attachment {
+	"""
+	Id
+	Unique identifier of the attachment
+	"""
+	id! string!
+
+	"""
+	Created At
+	UNIX timestamp (e.g. 1095379198.75) when the attachment was created
+	"""
+	createdAt! number!
```

Comment:
I am somewhat torn here, we use `createdAt` for ISO date fields in other profiles, so maybe use `createdTimestamp` instead?

Reply:
I think suffix `At` is general enough to be referencing to timestamp as well.

Reply:
It's also used in other capabilities in station.